### DISCUSSION TITLE
ver 0.146 AMBIENT AND HEATMAP IS NOT WORKING AGAIN on G513QM-HQ403TS 🤧

### DISCUSSION THREAD

**Discussion by RAGE-HERO7** [seerge/g-helper#1902] [General]
I don't wanna explain this same problem again bro😭 .so i am leaving the link to the previous post i mentioned this issue and you did send me a fixed ver (don't remember the ver no)that fixed my issue.

old discussion - https://github.com/seerge/g-helper/discussions/1781#discussion-5992434

The Ver you send me fixed AMBIENT,HEATMAP modes- https://github.com/seerge/g-helper/discussions/1781#discussioncomment-7937343

New log file - 
[log.txt](https://github.com/seerge/g-helper/files/13921799/log.txt)

picture to prove the issue (only light bar light's up my per key RGB supported keyboard won't turn on the light)- 
![20240112_215828](https://github.com/seerge/g-helper/assets/154586512/a6073ce5-52e9-47d5-aef8-7e616a2ad7ff)
![20240112_220610](https://github.com/seerge/g-helper/assets/154586512/9197cf2c-9dd0-41ef-a51b-31cc724fb8b2)

@seerge Help😭🙏

**Comment by seerge** [comment]
Closed as duplicate https://github.com/seerge/g-helper/discussions/1906

Please stop spamming topics in Discussions.

There is a separate "Issues" section for bug-reports
